test/integration/data-types.test.js

```diff
@@ -277,6 +278,16 @@ describe(Support.getTestDialectTeaser('DataTypes'), function() {
   it('should parse an empty GEOMETRY field', function () {
    var Type = new Sequelize.GEOMETRY();
 
+   // MySQL 5.7 or above doesn't support POINT EMPTY
+   if (dialect === 'mysql' && semver.gte(current.options.databaseVersion, '5.7.0')) {
+     return;
+   }
+
+   // Postgres 9.5.0 or above doesn't support POINT EMPTY
+   if (/^postgres/.test(dialect) && semver.gte(current.options.databaseVersion, '9.5.0')) {
```

Comment:
Rebased the docker pr onto this and its still failing. Not sure if this is a major deal, but this test still fails because any version of Postgis >=2.1.7 will fail this test. It really has nothing to do with the specific version of postgres. Not sure if we can test again postgis versions or not using sequelize? I am probably going to have to create a custom postgres/postgris docker image to get a version <= 2.1.7 because I have been unable to find one. If that is the case, we can ignore this and the custom docker image can just contain a version of postgres < 9.5.